### PR TITLE
CONTRIBUTING.md: Revise and clarify Copyright and PKG_MAINTANER fields.

### PR DESCRIPTION
**Pull request by thess**
Maintainer: @members
Description:

Here are a couple of long-overdue updates. Comments?

@neheb - could you add a sentence or 2 about choosing package source download locations and preferred archive formats. I can include it with this update.

Signed-off-by: Ted Hess \<thess@kitschensync.net\>

### CODE REVIEW

**Comment by neheb** [comment]
Maybe,

PKG_SOURCE should be the smallest available tarball. That is, xz > bzip2 > gz. Lzip is not supported.

PKG_SOURCE_URL should link to an official tarball of the package. If not available, a tarball generated remotely, like GitHub’s codeload should be used. And if that’s not available, a locally generated tarball using git or svn should be used. Convenience macros such as @SF, @APACHE, @GNU, and @GNOME are available.

The last part can be removed if that’s too wordy.

**Comment by thess** [comment]
Thanks Rosen - here's my additions/update.

**Comment by neheb** [comment]
Looks good
